### PR TITLE
Supported Gradients Accumulation

### PR DESCRIPTION
**Pull request by usimarit**

### CODE REVIEW

**Comment by ronggong** [comment]
@usimarit Does the gradient accumulation support multi gpus training. e.g mirrorStrategy?

**Comment by usimarit** [comment]
@ronggong Yes
